### ISSUE TITLE
Poll: Integrate iframe-steps into spreewald

### ISSUE THREAD

**Issue by judithroth** [makandra/spreewald#159]
In the weekly @makmic suggested to add iframe steps to spreewald. 
They look like this: [Card 1](https://makandracards.com/makandra/492360-how-to-test-inside-iframes-with-cucumber-capybara), [Card 2](https://makandracards.com/makandra/34015-use-capybara-commands-inside-an-iframe)

We decided against that in the past because not enough projects were using them.
Now, at least 6 of our projects use iframe steps. 

Please vote:
:+1: yes, add!
:-1: no, we don't need that yet

**Comment by triskweline** [comment]
Aesthetically I would prefer if the steps said "frame" instead of "iframe", since the cited steps already work with all kinds of frames.

Also an English speaker would use 1st, 2nd, etc. instead of 1., 2., etc.

**Comment by judithroth** [comment]
> Aesthetically I would prefer if the steps said "frame" instead of "iframe", since the cited steps already work with all kinds of frames.

I learned that that's not 100% correct for all of the steps. Frames and Iframes have different selectors (`frame` vs. `iframe`). Also I encountered strange problems with selection of frames (but not with the selection of iframes). Since frames are not relevant to us, we decided to not put effort into making it work for frames.

The new steps are available with Spreewald 4.1.0:

* `When ... inside the ... iframe`

New steps only available for Capybara 3+:

* `When I switch to the ... iframe`
* `When I switch back to the whole page`
